src/lib/biokepi.ml

```diff
@@ -0,0 +1,15 @@
+
+module Pipeline = Biokepi_pipeline_edsl.Pipeline
+
+module KEDSL = Biokepi_run_environment.Common.KEDSL
+
+module Metadata = Biokepi_run_environment.Metadata
+
+module Machine = Biokepi_run_environment.Machine
+
+module Setup = struct
```

Comment:
I actually started with more than one thing in there, and then I switched back to one module, but I wanted to see what `ocamldoc` does differently in both cases, and then I forgot :)

I can change it if you think the homogeneity is worth it. That module is going to change soon anyways, as I will figure out a way to generate better documentation.